src/Umbraco.Cms.Integrations.DAM.Aprimo/App_Plugins/UmbracoCms.Integrations/DAM/Aprimo/Render/AprimoAsset.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Umbraco.Cms.Integrations.DAM.Aprimo.Models.ViewModels.AprimoAssetViewModel>
+
+<link href="~/App_Plugins/UmbracoCms.Integrations/DAM/Aprimo/css/aprimo.css" rel="stylesheet"/>
+
+<div class="aprimo-container">
+    @if (!string.IsNullOrEmpty(Model.Thumbnail))
+    {
+        <img class="aprimo-thumbnail" src="@Model.Thumbnail" alt="@Model.Title" />
+    }
+    <h3 class="aprimo-title">@Model.Title</h3>
```

Comment:
I reckon this should probably be a `div`, as we can't know really what heading level is appropriate semantically here.  And the implementer can style this as a heading if they wanted to based on the CSS classes you have added.

Reply:
Updated, rendering also the cropped items.